Engine/src/sysFrame/System/File/File.cpp

```
@@ -0,0 +1,74 @@
+#include "File.h"
+
+#include <iostream>
+#include <Windows.h>
+
+bool File::Compress(const std::string& src, const std::string& dst) {
```

cppcheck (2.10-2)

[style] The function 'Compress' is never used.

(unusedFunction)